test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -25,6 +27,52 @@ describe('global navigation', () => {
     document.head.innerHTML = '<link rel="icon" href="/libs/img/favicons/favicon.ico" size="any"><script src="https://auth.services.adobe.com/imslib/imslib.min.js" type="javascript/blocked" data-loaded="true"></script>';
   });
 
+  describe('content source', () => {
+    const customPath = '/path/to/gnav';
+    let fetchStub;
+
+    beforeEach(() => {
+      fetchStub = sinon.stub(window, 'fetch');
+      sinon.stub(window.lana, 'log');
+      setConfig({ locale: { ietf: 'en-US', prefix: '' } });
+    });
+
+    afterEach(() => {
+      fetchStub = null;
+      window.fetch.restore();
+      window.lana.log.restore();
```

Comment:
A possible optimisation, that reduces the number of lines, would be to use `sinon.restore()` instead of individually restoring stubs.